activiti-api-impl/activiti-api-task-runtime-impl/src/main/java/org/activiti/runtime/api/impl/TaskAdminRuntimeImpl.java

```diff
@@ -88,6 +89,12 @@ public Task task(String taskId) {
         return new PageImpl<>(tasks,
                               Math.toIntExact(taskQuery.count()));
     }
+    
+    @Override
+    public Task update(UpdateTaskPayload updateTaskPayload) {
+        TaskUpdater taskUpdater= new TaskUpdater(taskService,taskConverter,null,null);
```

Comment:
@CTI777 let's instantiate `TaskUpdater` as a bean in `TaskRuntimeAutoConfiguration` and inject it via constructor here and in `TaskRuntimeImpl`.